server/auth/auth.interface.ts

```diff
@@ -16,6 +17,6 @@ export interface RequestWithPotentialUser<T = any> extends Request {
 }
 
 export interface RequestWithUser<T = any> extends Request {
-	user: UserDto;
+	user: UserModel;
```

Comment:
Since this thing is called at every request, I'd avoid constructing full `UserModel` and just rely on lightweight `UserDto` instead. 

That's the reason why `UserService` and `AuthService` were separated originally - we can use `AuthService` to do its own `getUserById` to be used here, and then just use the user domain model in `UserService.`

Reply:
resolved in https://github.com/mbrizic/ratethegame/pull/28/commits/b158c0718600a8adeb6bacdcd5d09c7e8ab913e4